lib/tasks/solution_path_cleanup.rake

```diff
@@ -0,0 +1,17 @@
+task :environment do
+  require File.expand_path('../../app',
+                           File.dirname(__FILE__)) # your Sinatra app
+end
+
+namespace :db do
+  desc "Make all solution paths relative to the problem directory, not root"
+  task solution_path_cleanup: :environment do
+    Submission.all.each do |submission|
```

Comment:
This should be `find_each`, otherwise we're pulling down almost 200k records at a time (and instantiating them as AR objects).